third_party/2and3/redis/connection.pyi

```diff
@@ -83,7 +86,7 @@ class Connection:
         encoding: Text = ...,
         encoding_errors: Text = ...,
         decode_responses: bool = ...,
-        parser_class: Optional[BaseParser] = ...,
+        parser_class: Optional[Type[BaseParser]] = ...,
```

Comment:
This is not `Optional` as far as I can see.

Reply:
right, updated